Framework/Muon/src/MuonAsymmetryHelper.cpp

```diff
@@ -115,7 +116,7 @@ size_t startIndexFromTime(const HistogramData::BinEdges &xData,
                           const double startX) {
   auto upper =
       std::lower_bound(xData.rawData().begin(), xData.rawData().end(), startX);
-  return std::distance(xData.rawData().begin(), upper + 1);
+  return std::distance(xData.rawData().begin(), upper);
```

Comment:
@TWJubb is this the change we discussed in #https://github.com/mantidproject/mantid/issues/22465?

Reply:
It is not tested yet, but I *think* this is the fix, yes. Perhaps worth testing as part of fixing #22465. Several unit tests went down from this change; so there is a fair amount of work to fix.

Reply:
thank you Tom. I think it might be best to change it back here and then make the change in the other issue. To keep the changes isolated.